diagrams/initial-api-schema-proposal.json

```diff
@@ -0,0 +1,100 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "API Title",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.server.test/v1"
+    }
+  ],
+  "paths": {},
+  "components": {
+    "schemas": {
+      "form": {
+        "type": "object",
+        "properties": {
+          "id": {
```

Comment:
Do we need a version in Form as well as the openapi version? It might make it easier to iterate if the runner is able to serve multiple versions of the form structure at the same time. It might also make it more complicated and isn't something we need yet.